1 Standard/007_Squirtle.yaml

```diff
@@ -35,3 +35,22 @@ ball: [Poke]
 nature: Adamant
 evs: {hp: 184, atk: 252, def: 36, spA: 0, spD: 36, spe: 0}
 ivs: 31
+---
+species: Squirtle
+setname: Project M
+tags: [standard]
+item: [Null]
```

Comment:
`null`, not `Null`

Reply:
Weird, but the other Squirtle movesets also had capital N-Nulls in their movesets before I even touched it. Whatever, changing.